sssd_test_framework/utils/sssctl.py

```diff
@@ -90,3 +93,28 @@ def passkey_register(
             )
 
         return result.stdout_lines[-1].strip()
+
+    def cache_expire(self, *args: str) -> SSHProcessResult:
```

Comment:
I just added this in another pull request as I needed it to test a netgroup bug. Please, remove this one.